documentation/news/2018-state-of-streisand.md

```diff
@@ -0,0 +1,120 @@
+
+Special thanks to @alimakki for laying the translation groundwork and for the FR translation, to @levleontiev for the RU translation, and to @HougeLangley for the CN translation.
+
+### Modularity
+
+A persistent point of feedback about the Streisand project has been that it installs a “kitchen sink” of software whose combined threat surface is unmanageable. Often this perspective is held by individuals with a different threat model & end goal in mind than what Streisand intends to address. There is truth in the criticism but the real world is seldom black and white.
+
+Ultimately a diversity of tactics & technology is required to meet different user needs. Users on hostile networks often do not know apriori which software/options will work for their environment on any given day, and the censorship landscape shifts frequently and unexpectedly. A principled stance that only offers technology tailored to unfiltered North American internet connections does little for more marginalized users.
+
+To strike a balance that also acknowledges use-cases that would prefer to deploy a VPN server with a reduced threat surface, we’ve completed [a large and intensive rework of the core VPN services in Streisand to support modularity](https://github.com/StreisandEffect/streisand/issues/746). It is now possible to pick and choose which services are deployed on a Streisand server. Deployed roles can be customized interactively at install time or pre-selected and used for many deployments using a single  site configuration file.
+
+### New Provisioning Modes
+
+In the four years since Streisand was announced the project has seen users adapt the included Ansible roles for a number of use-cases that weren’t met by the core project. One of the earliest examples of this was adapting Streisand to provision the same computer that Streisand is run on, instead of a new remote server. This deployment model worked well for users that couldn’t install Ansible on their laptops or who preferred having a self-contained Ubuntu 16.04 server that would bootstrap itself with Streisand.
+
+Inspired by these creative modifications we’re happy to announce that we have added several new “advanced” deployment modes:
+
+Localhost - Use Streisand to set up the machine it’s running on as a VPN server.
+Existing Server - Use Streisand to set up an existing Ubuntu 16.04 cloud server. This allows Streisand to run on servers from a variety of cloud providers that Streisand does not support out of the box (It works great with [Vultr](https://www.vultr.com/) for example!).
+Non-Interactive - Use Streisand without having to answer any prompts by editing a configuration file in advance. This can be great for programmatic or container-based usage. Thanks to @exratione for this feature.
+
```

Comment:
I think this section would look better if it was bulleted for each section, for example:

```
Localhost
 -  Use Streisand to set up the machine it’s running on as a VPN server
```

Reply:
Good idea - I think I lost some formatting here in the translation from Google Docs. Fixed in e932c13